homeassistant/components/homeassistant_sky_connect/__init__.py

```diff
@@ -24,19 +39,44 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
         # The USB dongle is not plugged in
         raise ConfigEntryNotReady
 
-    usb_info = usb.UsbServiceInfo(
-        device=entry.data["device"],
-        vid=entry.data["vid"],
-        pid=entry.data["pid"],
-        serial_number=entry.data["serial_number"],
-        manufacturer=entry.data["manufacturer"],
-        description=entry.data["description"],
-    )
+    addon_manager: AddonManager = get_addon_manager(hass)
+    try:
+        addon_info: AddonInfo = await addon_manager.async_get_addon_info()
```

Comment:
How will this work for container and core installations that do not have the supervisor ?

Reply:
Fixed